fvm/src/executor/default.rs

```diff
@@ -104,16 +104,16 @@ where
                 }
             }
             Err(ExecutionError::OutOfGas) => Receipt {
-                exit_code: ExitCode::SysErrOutOfGas,
+                exit_code: SystemExitCode::OUT_OF_GAS,
                 return_data: Default::default(),
                 gas_used,
             },
             Err(ExecutionError::Syscall(err)) => {
                 let exit_code = match err.1 {
-                    ErrorNumber::IllegalOperation => ExitCode::SysErrIllegalActor,
-                    ErrorNumber::AssertionFailed => ExitCode::SysErrIllegalArgument,
-                    ErrorNumber::InsufficientFunds => ExitCode::SysErrInsufficientFunds,
-                    ErrorNumber::NotFound => ExitCode::SysErrInvalidReceiver,
+                    ErrorNumber::IllegalOperation => SystemExitCode::ILLEGAL_ACTOR,
```

Comment:
Here is where I need ILLEGAL_ACTOR (dropped by the spec), or an alternative.

Reply:
Send _shouldn't_ be able to return this, I think. If it can, I believe the correct exit code would likely be `SysErrInvalidReceiver`.

Send should only return:

https://github.com/filecoin-project/fvm-specs/blob/724650bf92b239bb7d2079cde7d5bad121d7c79f/07-errors.md#sendsend

Reply:
I'll remove the case for now (so it becomes an unhandled error) so I can drop ILLEGAL_ACTOR. I'll then follow-up to address all possible error numbers explicitly. #438.